src/rovr_control/rovr_control/main_control_node.py

```diff
@@ -21,7 +21,7 @@
 import asyncio  # Allows the creation of asynchronous autonomous procedures!
 import subprocess  # This is for the webcam stream subprocesses
 import signal  # Allows us to kill subprocesses
-import serial  # Serial communication with the Arduino. Install with: <sudo pip3 install pyserial>
+import serial  # Allows serial communication with an Arduino. Install with "sudo pip3 install pyserial".
```

Comment:
Why leave this import if it is unused

Reply:
Fair, I wanted to leave this as a reference in case we need it later, but since this is in our archive release we should be fine to delete it from main.